client/app/views/steps/subviews/progression.coffee

```diff
@@ -0,0 +1,4 @@
+{LayoutView} = require 'backbone.marionette'
+
+module.exports = class ProgressionView extends LayoutView
```

Comment:
The very first view was a copy-paste from another one, and LayoutView stayed.

I am changing it right now.